src/routes/users/components/UpdateUserForm.tsx

```diff
@@ -103,7 +105,7 @@ const UpdateUserForm = ({
                 status: 'success',
                 isClosable: true,
             });
-
+            fetchUsersList();
```

Comment:
If you manually refresh the list here, then there's no need to update the list in useUsersStore in the updateUser method.

Reply:
That's when deleting user not updating :)

Reply:
My bad, I didn't look at the immediate context :(